lib/index.js

```diff
@@ -71,6 +71,9 @@ export default function runContentfulExport (usageParams) {
           }
         )
       }
+      return response
```

Comment:
Need to do that anyways for proper progress indication in the next sprint. But out of scope IMHO for this PR